pandas/tests/plotting/test_frame.py

```diff
@@ -538,7 +538,8 @@ def test_subplots_timeseries_y_axis(self):
             ax_datetime_no_tz.get_lines()[0].get_data()[1]
             == testdata["datetime_no_tz"].values
         ).all()
-        ax_datetime_all_tz = testdata.plot(y="datetime_all_tz")
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
is this the same warning filtering as above?

can you add a comment about what issue this is attached

Reply:
What do you mean by "the same"?

> can you add a comment about what issue this is attached

Do you mean [this](https://github.com/pandas-dev/pandas/issues/30334)? It's mentioned in the PR. description.

Reply:
meaning its completely non-obvious which FutureWarning you are test, so either use a regex or add a comments

Reply:
PR descriptions are not very useful once the code is pushed, only in the review. comments are much preferred..